x/rewards/keeper/grpc_query.go

```diff
@@ -69,11 +69,14 @@ func (s *QueryServer) BlockRewardsTracking(c context.Context, request *types.Que
 	ctx := sdk.UnwrapSDKContext(c)
 	height := ctx.BlockHeight()
 
-	blockRewards, found := s.keeper.state.BlockRewardsState(ctx).GetBlockRewards(height)
-	if !found {
+	blockRewards, err := s.keeper.BlockRewards.Get(ctx, uint64(height))
+	if err == nil {
```

Comment:
If I read this correctly it means there simply aren't inflationary rewards if there are no records found;
i.e. an error here indicates not found, thus no inflationary rewards...

Should we add a comment to make this clear?
e.g. // err != nil -> "not found", thus no inflationary rewards